azure-pipelines.yml

```diff
@@ -24,6 +24,12 @@ variables:
   DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
 
 steps:
+- task: UseDotNet@2
+  displayName: 'Use .NET Core sdk'
+  inputs:
+    packageType: sdk
+    version: 3.0.100
```

Comment:
I'd use '3.0.x' as the version as that ensures you always use the latest patch release (fixes/security updates)